server_go_not111.go

```diff
@@ -0,0 +1,23 @@
+// +build !go1.11 windows
+
+package dns
+
+import "net"
+
+const supportsReuseport = false
+
+func listenTCP(network, addr string, reuseport bool) (net.Listener, error) {
+	if reuseport {
+		// TODO(tmthrgd): return an error?
```

Comment:
@miekg What would your preferred behaviour here be? Ignore the reuseport bool or return an error?

Reply:
on the fence..., but... ignore?

Reply:
Ack.